### PR TITLE
Nabil Abdel-Rahman

### PR DESCRIPTION
**Pull request by nabilabdel-rahman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,66 +1,17 @@
...
-* If the test cases are missing/not functional/do not all pass, 5 points will be docked off per each (25 points total).
-
-* If `README` is not clear, missing or not edited, or `Dockerfile` is not updated, up to 15 points will be docked off.
-
-* If none of the functionalities work, 10 points will be given assuming `credentials.ini` is submitted with the correct repo URL, and `Dockerfile` builds and runs without any errors. 
-
-## Authors
-
-Michal Young, Ram Durairajan. Updated by Ali Hassani.
+Make sure to run the commands docker stop and to delete the container and image when finished running the program.
```
Comment (MilesAnderson) [inline README.md:17]: Really good really detailed README

File: brevets/acp_times.py [hunk 2]
```diff
@@ -13,6 +13,8 @@
 #  same arguments.
 #
 
+MAX_SPEEDS = [34, 32, 30, 28, 28, 26]
+MIN_SPEEDS = [15, 15, 15, 11.428, 11.428, 13.333]
```
Comment (MilesAnderson) [inline brevets/acp_times.py:17]: I was wondering how to do this project without a bunch of if statements but didn't have the motivation to figure it out, so it's cool to see it implemented here.

File: brevets/acp_times.py [hunk 3]
```diff
@@ -41,4 +58,33 @@ def close_time(control_dist_km, brevet_dist_km, brevet_start_time):
        An arrow object indicating the control close time.
        This will be in the same time zone as the brevet start time.
     """
-    return arrow.now()
+    i = 0
+    time = 0
+    additionalTime = 0.5
```
Comment (MilesAnderson) [inline brevets/acp_times.py:63]: I'm having trouble seeing why you set this to 0.5 instead of 0.

File: brevets/acp_times.py [hunk 4]
```diff
@@ -26,7 +28,22 @@ def open_time(control_dist_km, brevet_dist_km, brevet_start_time):
...
+    if control_dist_km > brevet_dist_km:
+       control_dist_km = brevet_dist_km
+
+    while control_dist_km > 200:
+        time += (200/MAX_SPEEDS[i])
+        i += 1
+        control_dist_km -= 200
+
+    time += (control_dist_km/MAX_SPEEDS[i])        
+    time = ((time * 60) + 0.5)
```
Comment (MilesAnderson) [inline brevets/acp_times.py:43]: As far as I can tell the '+ 0.5' works but you can also always use the python round() function.

**Review by MilesAnderson — COMMENTED** [review]
Really well done. The README was very thorough. The acp_times.py was done very well, you used significantly less lines than I did and I feel like in many ways it was more readable. Everything else was done well, I don't feel like there is much to comment on when it comes to the flask and HTML files. Your tests were also well written and covered some important edge cases. 
The only critique I have is the lack of comments. It wasn't a big deal in this project since there wasn't a lot of code and it was all pretty straightforward. But I think it is a good habit to add clarifying comments around your code. All in all well done.
